playbooks/troubleshooting/troubleshooting_guide.adoc

```diff
@@ -0,0 +1,719 @@
+    info, err := os.Stat(a.file)
+    if err != nil {
+        return err
+    }
+
+    if a.fileInfo == nil \|\| a.fileInfo.ModTime() != info.ModTime() {
+        glog.V(4).Infof("Loading htpasswd file %s...", a.file)
+        loadingErr := a.load()
+        if loadingErr != nil {
+            return err
+        }
+
+        a.fileInfo = info
+        return nil
+    }
+    return nil
+}
+----
+
+==== Solution:
+
+Create the htpasswd on each master, or otherwise sync the correct
+htpasswd file
+ +
+ +
+
+== Misc
+
+=== Docker won't start
+
+----
+[root@master00 ~]# systemctl status docker
+● docker.service - Docker Application Container Engine
+   Loaded: loaded (/usr/lib/systemd/system/docker.service; enabled; vendor preset: disabled)
+   Active: failed (Result: start-limit) since Wed 2016-03-02 20:12:43 CST; 8s ago
+     Docs: http://docs.docker.com
+  Process: 2577 ExecStart=/usr/bin/docker daemon $OPTIONS $DOCKER_STORAGE_OPTIONS $DOCKER_NETWORK_OPTIONS $ADD_REGISTRY $BLOCK_REGISTRY $INSECURE_REGISTRY (code=exited, status=1/FAILURE)
+ Main PID: 2577 (code=exited, status=1/FAILURE)
+
+Mar 02 20:12:43 master00 systemd[1]: docker.service: main process exited, code=exited, status=1/FAILURE
+Mar 02 20:12:43 master00 systemd[1]: Failed to start Docker Application Container Engine.
+Mar 02 20:12:43 master00 systemd[1]: Unit docker.service entered failed state.
+Mar 02 20:12:43 master00 systemd[1]: docker.service failed.
+Mar 02 20:12:43 master00 systemd[1]: docker.service holdoff time over, scheduling restart.
+Mar 02 20:12:44 master00 systemd[1]: start request repeated too quickly for docker.service
+Mar 02 20:12:44 master00 systemd[1]: Failed to start Docker Application Container Engine.
+Mar 02 20:12:44 master00 systemd[1]: Unit docker.service entered failed state.
+Mar 02 20:12:44 master00 systemd[1]: docker.service failed.
+----
+
+Check */var/log/messages*
+
+----
+Mar  2 20:06:43 master00 docker: time="2016-03-02T20:06:43.672735546-06:00" level=info msg="Listening for HTTP on unix (/var/run/docker.sock)"
+Mar  2 20:06:43 master00 docker: time="2016-03-02T20:06:43.873012061-06:00" level=warning msg="Docker could not enable SELinux on the host system"
+Mar  2 20:06:43 master00 docker: time="2016-03-02T20:06:43.879826788-06:00" level=fatal msg="Error starting daemon: Error loading key file /etc/docker/key.json: unable to decode private key JWK: decoding JWK Private Key JSON data: unexpected end of JSON input\n"
+----
+
+==== Solution:
+
+Chances are the key is empty. Remove the key and restart docker, the key
+will be regenerated
+ +
+ +
+
+=== oc new-app runs s2i instead of Docker build
+
+An application that was created containing a builder image appears to
+ignore any Dockerfile that is in the github repo. To ensure a Docker
+build occurs instead of an s2i build, the application only needs to be
+created with a github repo. 
+
+----
+oc new-app https://github.com/lawnjarae/eap-openshift-rhc-license.git
```

Comment:
Currently the build in OSE fails due to the incorrect image name (image not found error). The base image name on the FROM line in the Dockerfile of this repo has an outdated name.  I have submitted a PR [1] against that repo to have the change made or point to a repo that has an updated image name. 

[1] https://github.com/JaredBurck/eap-openshift-rhc-license/pull/1